tests/test_designmatrix_noisepars.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pint.models.model_builder as mb
+import pinttestdata
+import pytest
+
+
+@pytest.fixture()
+def model_and_toas():
+    datadir = pinttestdata.datadir
+    parfile = os.path.join(datadir, "J1713+0747_NANOGrav_11yv0_short.gls.par")
```

Comment:
Does this par file have the noise parameters unfrozen? What makes it special? Any reason not to use a simpler one with a set of TOAs that is much faster to load?

Reply:
I just picked one which had EFACs, EQUADs, and ECORRs.

Reply:
Can you create a really simple one in-line? This test is likely to take unnecessarily long if you use this very large data set. It would also be best if it were obvious what is needed from the par file.

Reply:
OK. I will create a smaller dataset for this.